docs/monitoring/faq/webhooks.mdx

```diff
@@ -77,3 +77,116 @@ When delivering events to your webhook endpoint we follow these guidelines
 - If your endpoint returns a 5xx status code in less than 5 seconds we retry up to 2 times with exponential backoff.
 - If your endpoint returns a 4xx status code, we declare the delivery failed and do not retry.
 - If your endpoint returns any body do not read it
+
+## Example with Modal
+
+### Setup
+
+For an example of how to set this up, we will use [Modal](https://modal.com/).
```

Comment:
Suggested extra copy:

"Modal provides autoscaling GPUs for inference and fine-tuning, secure containerization for code agents, and serverless Python web endpoints. We'll focus on the web endpoints here."